### ISSUE TITLE
ADA compliance

### ISSUE THREAD

**Issue by jaymollica** [ic-labs/glamkit-alerts#1]
Right now the alerts are nested in the `<nav>`, this is misleading to a screen reader.  Only navigational items should be nested in `<nav>`.  A good step forward would be to put it outside `<nav>` and make visible elements hidden from screen readers, then create a new element that is offscreen (ie -9999px to the left) that a screen reader would pick up on.

```
<div class="screen-reader-alert-container">
<h1>Tips for visiting SFMOMA today</h1>
<ul>
<li>Alert 1</li>
<li>Alert 2</li>
...
</ul>
</div>
```
